src/pages/events.js

```diff
@@ -169,14 +163,15 @@ function EventsPage({data}) {
         <section className="wrapper-carousel style5">
           <div className="inner-carousel">
             <Carousel loop widgets={[IndicatorDots, CarouselArrows]}>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel1}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel2}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel3}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel4}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel5}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel6}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel7}/>
-              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel8}/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel1} alt='photocredits to Conta-me Como Foi' title='photocredits to Conta-me Como Foi'/>
+
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel2} alt='photocredits to João Makes Photos' title='photocredits to João Makes Photos'/>
+
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel3} alt='photocredits to Saiotes' title='photocredits to Saiotes'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel4} alt='photocredits to Madalena Abreu' title='photocredits to Madalena Abreu'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel5} alt='photocredits to Pedro Marcelino' title='photocredits to Pedro Marcelino'/>
+              <BackgroundImage Tag="div" className="carousel-image" fluid={carousel6} alt='photocredits to Pedro Marcelino' title='photocredits to Pedro Marcelino'/>
+              
               <BackgroundImage Tag="div" className="carousel-image" fluid={carousel9}/>
```

Comment:
Will probably leave the last picture as it's not worth it to wait for one single image to be chosen.